hack/install-ocs-client.sh

```diff
@@ -16,5 +16,17 @@ fi
 
 "$OPERATOR_SDK" run bundle "$OCS_CLIENT_BUNDLE_FULL_IMAGE_NAME" --timeout=10m --security-context-config restricted -n "$INSTALL_NAMESPACE" --index-image "$CSI_ADDONS_CATALOG_FULL_IMAGE_NAME"
 
+# Ensure position independent make targets in release CI, explicitly setting the values ensures client-op doesn't deploy CSI
+# when storagecluster is configured for remoteconsumers, controllers set this value to "true"
+cat <<EOF | oc create -f -
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ocs-client-operator-config
+  namespace: openshift-storage
+data:
+  DEPLOY_CSI: "false"
```

Comment:
@leelavg Who is responsible for creating this configmap in the actual cluster?

If it is going to be ocs-operator what if ocs-operator comes up after ocs-client-operator?

Reply:
this is created as part of ocs-client-op bundle.

Reply:
So who will change its value when only ocs-client-operator will be deployed?

Reply:
only ocs-client-operator will not be deployed when run in CI for ocs-operator. When ocs-client-operator is alone deployed it looks as StorageCluster CRD to deploy CSI or not, however during CI we can't guarantee which comes first and so this is only an insurance.